src/hooks/useGameData.js

```diff
@@ -1,13 +1,13 @@
 import {useState} from 'react'
 import { v4 as uuid } from 'uuid'
-import saveLocally from '../lib/saveLocally'
-import loadlocally from '../lib/loadLocally'
+import SaveLocally from '../lib/SaveLocally'
+import Loadlocally from '../lib/LoadLocally'
 
 export default function useGameData() {
 
-    const [savedGameProfiles, setSavedGameProfiles] = useState(loadlocally('gameProfiles') ?? [])
+    const [savedGameProfiles, setSavedGameProfiles] = useState(Loadlocally('gameProfiles') ?? [])
     
-    saveLocally('gameProfiles', savedGameProfiles)
+    SaveLocally('gameProfiles', savedGameProfiles)
```

Comment:
Any Reason you started the "localStorage functions" with a Capital Letter? Just Curios..

Reply:
It was a misunderstanding for my part. :D